backend/backend/src/main/java/de/aaaaaaah/velcom/backend/newaccess/committaccess/CommitReadAccess.java

```diff
@@ -231,7 +231,8 @@ public FullCommit getFullCommit(RepoId repoId, CommitHash commitHash)
 
 		try (DBReadAccess db = databaseStorage.acquireReadAccess()) {
 			SelectConditionStep<KnownCommitRecord> query = db.selectFrom(KNOWN_COMMIT)
-				.where(KNOWN_COMMIT.REPO_ID.eq(repoId.getIdAsString()));
+				.where(KNOWN_COMMIT.REPO_ID.eq(repoId.getIdAsString()))
+				.and(KNOWN_COMMIT.TRACKED.isTrue());
```

Comment:
We use `0` and `1` as boolean values but `isTrue` checks against string literals.